packages/shared/src/porter.ts

```diff
@@ -17,36 +17,60 @@ import { Base64EncodedBytes, ChecksumAddress, HexEncodedBytes } from './types';
 import { fromBase64, fromHexString, toBase64, toHexString } from './utils';
 
 const defaultPorterUri: Record<string, string> = {
-  mainnet: 'https://porter.nucypher.community',
-  tapir: 'https://porter-tapir.nucypher.community',
-  lynx: 'https://porter-lynx.nucypher.community',
+  mainnet: 'https://porter.nucypher.io',
+  tapir: 'https://porter-tapir.nucypher.io',
+  lynx: 'https://porter-lynx.nucypher.io',
 };
 
+const porterUriSource: string =
+  'https://raw.githubusercontent.com/nucypher/nucypher-porter/development/porter_instances.json';
```

Comment:
Currently, this file contains the same values as the defaults...should the file in the repos be an empty dict for the time being, and we add to it over time, if there are other non-default porter URIs that can be used? i.e. the functionality is there if non-default porter URIs become available to use.

Reply:
I thought maybe file in repo will be empty, but with second thought now - maybe just check array for duplicates?

Reply:
Any reason you don't want the repos file to be empty?

Reply:
No reasons, I'm totally fine with that

Reply:
Cool. Let's update the file in `nucypher-porter` to be empty then. If there are other non-default porter URIs we can use, we can add them at that time. 

I assume the code in this PR can handle an empty dict.

Reply:
Followed-up in https://github.com/nucypher/nucypher-porter/pull/76